deploy/scripts/src/cri-lime/test.ts

```diff
@@ -163,13 +165,33 @@ const profiles: ProfileList = {
 }
 
 const loadProfile = selectProfile(profiles)
+type drivingLicenceIssuer = 'DVA' | 'DVLA'
+const licenceIssuer: drivingLicenceIssuer = (Math.random() <= 0.5) ? 'DVA' : 'DVLA'
+const groupMap = {
+  fraud: [
+    'B01_Fraud_01_CoreStubEditUserContinue',
+    'B01_Fraud_02_ContinueToCheckFraudDetails',
+    'B01_Fraud_02_ContinueToCheckFraudDetails::01_CRICall',
+    'B01_Fraud_02_ContinueToCheckFraudDetails::02_CoreStubCall'
+  ],
+  drivingLicence: [
+    'B02_Driving_01_DLEntryFromCoreStub',
+    `B02_Driving_02_Select${licenceIssuer}`,
+    `B02_Driving_03_${licenceIssuer}_EnterDetailsConfirm`,
+    `B02_Driving_03_${licenceIssuer}_EnterDetailsConfirm::01_CRICall`,
+    `B02_Driving_03_${licenceIssuer}_EnterDetailsConfirm::02_CoreStubCall`
```

Comment:
This is only going to generate thresholds for one of `DVA` or `DVLA`, it needs to be both

Reply:
Yeah did not realise that I moved this in the init stage instead of the virtual user code section. This has now been updated accordingly.